invenio_app_ils/ui/src/invenio_app_ils/pages/frontsite/Home/Home.js

```diff
@@ -29,53 +23,77 @@ export default class Home extends Component {
 
   render() {
     return (
-      <div className="home-container">
-        <Grid centered columns={2}>
+      <Grid>
+        <Grid.Row centered columns={1}>
           <Header size="huge">CERN Library</Header>
-
-          <Grid.Row centered columns={2}>
-            <Container className="books-search-searchbar">
-              <Form onSubmit={this.onSubmit}>
-                <Input
-                  fluid
-                  icon={
-                    <Icon
-                      name="search"
-                      inverted
-                      circular
-                      link
-                      onClick={this.onSubmit}
-                    />
-                  }
-                  size="large"
-                  query={this.state.query}
-                  onChange={this.updateQuery}
-                  placeholder="Search for books, articles, proceedings..."
+          {/* TODO: This can be replaced by the common component SearchBar */}
```

Comment:
Please fix or if really needed ticketize any TODO, we will forget for sure them...